packages/multichain-transactions-controller/src/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { BtcAccountType, SolAccountType } from '@metamask/keyring-api';
+
+/**
+ * The network identifiers for supported networks in CAIP-2 format.
+ * Note: This is a temporary workaround until we have a more robust
+ * solution for network identifiers.
+ */
+export enum MultichainNetworks {
```

Comment:
Nit: We can think of an enum as a type union, and it's conventional to use a singular name for type unions rather than a plural name. What about:

``` suggestion
export enum MultichainNetwork {